pkg/client/client.go

```diff
@@ -0,0 +1,35 @@
+package client
+
+import (
+	"github.com/oracle/oci-native-ingress-controller/pkg/certificate"
+	"github.com/oracle/oci-native-ingress-controller/pkg/loadbalancer"
+	"github.com/oracle/oci-native-ingress-controller/pkg/waf"
+	"k8s.io/client-go/kubernetes"
+)
+
+type Client struct {
```

Comment:
Can we modify the name of this struct to ClientProvider?

Reply:
Done